Bottom: e595827e0b21a6e6f34428119ebbd6b0d3ddb2bf
Top:    b3c1d7a6f3a6760afdb78409535dd2c442776075
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-05-11 06:33:24 -0300

Refresh of dependent-special-funcs.patch

---

diff --git a/include/gcc-cp-fe.def b/include/gcc-cp-fe.def
index ab09ff8..6ba208c 100644
--- a/include/gcc-cp-fe.def
+++ b/include/gcc-cp-fe.def
@@ -519,6 +519,8 @@ GCC_METHOD3 (gcc_expr, type_value_expr,
 /* Return the type of the gcc_expr OPERAND.
    Use this for decltype.
 
+   FIXME: introduce decltype (auto).
+
    Note: for template-dependent expressions, the result is NULL,
    because the type is only computed when template argument
    substitution is performed.  */
